etrago/tools/plot.py

```diff
@@ -1357,15 +1498,15 @@ def plot_grid(self,
     if disaggregated:
         network = self.disaggregated_network.copy()
     else:
-        network = self.network.copy()
+        network = self.copy()
```

Comment:
I always run into errors with this change. Since `self` is an Etrago-object, the PyPSA-network is stored in `self.network`. Could you explain why you changed it?

Reply:
normalized again in 038a0df6563d21066d764ec7a663ee2bd4dad362